test/customRules-pre-imported/rules/any-blockquote.js

```diff
@@ -0,0 +1,26 @@
+// @ts-check
```

Comment:
Another exact copy, right?

Reply:
Ditto for all the test files.

Reply:
For the most part yes, since this is testing the ability to process the object/function directly, I removed all but one rule for the custom rules (copy of `test/customRules/`) and all but one output for the output formatters (copy of `test/outputFormatters-params/`).